adapters/appnexus/appnexus.go

```diff
@@ -281,14 +310,20 @@ func splitRequests(imps []openrtb2.Imp, request *openrtb2.BidRequest, requestExt
 	headers.Add("Content-Type", "application/json;charset=utf-8")
 	headers.Add("Accept", "application/json")
 
-	var err error
-	request.Ext, err = json.Marshal(requestExtension)
+	appnexusExtJson, err := json.Marshal(requestExtAppnexus)
 	if err != nil {
 		errs = append(errs, err)
 	}
 
-	for impsLeft {
+	requestExtClone := maputil.Clone(requestExt)
+	requestExtClone["appnexus"] = appnexusExtJson
```

Comment:
Why do we need the `requestExtClone`? We don't seem to do anything with `requestExt` and after exiting the `splitRequests` function, `requestExt` goes immediately out of scope

Reply:
`splitRequests` is called in a loop by `buildAdPodRequests`. However, it just sets the `appnexus` adapter field and then marshals. Although the `requestExtAppnexus` is different in each iteration, it is always set.. so perhaps this doesn't need to clone? Thoughts given this context?